src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.Unix.cs

```diff
@@ -19,140 +19,135 @@ namespace System.Net.Security
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal class NegoState
+    internal partial class NegoState
```

Comment:
In other System.Net projects, I believe the practice has been to name types like this with a Pal suffix, e.g. NegotiateStatePal (I'd also prefer it if we expanded out the name like that).